File: RLE.API/Models/Region.cs
using System;
using System.Collections.Generic;

namespace RLE.API.Models
{
    public partial class Region
    {
        

        public int Id { get; set; }
        public string Name { get; set; }
        public int? DistrictId { get; set; }
        public string Code { get; set; }
        public byte Active { get; set; }
        public bool ActiveforInscription { get; set; }

        public  District District { get; set; }
        public virtual ICollection<Department> Departments { get; set; }
        // public virtual  ICollection<User> Employee { get; set; }
        // public virtual ICollection<Training> Training { get; set; }
        // public virtual ICollection<TrainingClass> TrainingClass { get; set; }
        // public  ICollection<Vehicle> Vehicle { get; set; }
    }
}

File: RLE-SPA/src/app/_models/region.ts
import { Department } from './department';

export class Region {
  public id: number;

  public name: string;

  public districtId: number;

  public code: string;
  totalSelected : number;
  totalReserved : number;
  public active: number;
  totalOnTraining: number;
  totalPreSelected: number;
  totalRegistered: number;
  nbEmpNeeded: number;
  prctRegistered: number;
  prctPreselected: number;
  prctOnTraining: number;
  prctSelected: number;
  activeforInscription: boolean;
  departments: Department[];


}
